From 4184c4fd7685e637db2f9f1b0442f9c15a3dee0c Mon Sep 17 00:00:00 2001
From: Tiago Freire <41837236+tiagofreire-pt@users.noreply.github.com>
Date: Wed, 24 Jan 2024 10:03:03 +0000
Subject: [PATCH] Harmonizing the improvement on backup-switchover-mode overlay
 value definitions

On the followup of https://github.com/raspberrypi/linux/pull/5884, I missed a second duplicate definition. Now, harmonized the entire document.

Signed-off-by: Tiago Freire <41837236+tiagofreire-pt@users.noreply.github.com>
---
 arch/arm/boot/dts/overlays/README | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/README b/arch/arm/boot/dts/overlays/README
index d4de5b8eb9b9..6fa3cf20f287 100644
--- a/arch/arm/boot/dts/overlays/README
+++ b/arch/arm/boot/dts/overlays/README
@@ -2190,7 +2190,13 @@ Params: abx80x                  Select one of the ABx80x family:
                                 source
 
         backup-switchover-mode  Backup power supply switch mode. Must be 0 for
-                                off or 1 for Vdd < VBackup (RV3028, RV3032)
+                                "Switchover disabled", 1 for "Direct Switching"
+                                (if Vdd < VBackup), 2 for "Standby
+                                Mode" (if Vdd < Vbackup,
+                                does not draw current) or 3 for
+                                "Level Switching" (if Vdd < Vbackup
+                                and Vdd < Vddsw and Vbackup > Vddsw)
+                                (RV3028, RV3032)
 
         i2c_gpio_sda            GPIO used for I2C data (default "23")
 
-- 
2.34.1

